test/Microsoft.TemplateEngine.TestHelper/PackageManager.cs

```diff
@@ -265,7 +309,7 @@ private IEnumerable<PackageSource> LoadNuGetSources(params string[] additionalSo
                 throw new Exception($"Failed to load NuGet sources configured for the folder {currentDirectory}", ex);
             }
 
-            if (!additionalSources?.Any() ?? true)
+            if (additionalSources == null || !additionalSources.Any())
```

Comment:
I would assume that it is guaranteed that `additionalSources` can't be `null` here, since this is private method, and `#nullable enable` is turned on?